apiserver/params/storage.go

```diff
@@ -33,3 +33,28 @@ type StorageShowResult struct {
 type StorageListResult struct {
 	Instances []StorageInstance
 }
+
+// PoolInstance holds data for a pool instance.
+type PoolInstance struct {
+	Name   string
+	Type   string
+	Traits map[string]interface{}
```

Comment:
I'm not married to the name "config", but I think we should be consistent or we'll have a hard time talking about things as a team. Can you please call it config, and then we can talk about renaming it all to something else?